src/linker/Linker/LinkerErrorException.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Mono.Linker
+{
+	public class LinkerErrorException : Exception
```

Comment:
I'd suggest to name it `LinkerFatalErrorException` or something like that to make it clear it's only for unrecoverable situations